lib/pure/terminal.nim

```diff
@@ -462,15 +509,35 @@ elif not defined(windows):
     result = stdin.readChar()
     discard fd.tcsetattr(TCSADRAIN, addr oldMode)
 
+# Deprecated procs assuming output to stdout:
+template setCursorPos*(x, y: int) {.deprecated.} = setCursorPos(stdout, x, y)
```

Comment:
If my program uses terminal.nim and I was happy with it using stdout because it happens *not* to be a compiler, why force me to write ``stdout.setCursorXPos(...)`` etc all the time? Only if I actually want to move from stdout to stderr there should be an option like -d:nimWarnAboutStdout so that it's easy to update my code.